sdk/eventhubs/azure-eventhubs/src/samples/java/com/azure/messaging/eventhubs/eventprocessor/InMemoryPartitionManager.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.eventprocessor;
+
+
+import com.azure.messaging.eventhubs.PartitionManager;
+import com.azure.messaging.eventhubs.eventprocessor.models.Checkpoint;
+import com.azure.messaging.eventhubs.eventprocessor.models.PartitionOwnership;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * A simple in-memory implementation of a {@link PartitionManager}
+ */
+public class InMemoryPartitionManager implements PartitionManager {
+
+    private Map<String, PartitionOwnership> partitionOwnershipMap = new ConcurrentHashMap<>();
+
+    @Override
+    public Flux<PartitionOwnership> listOwnership(String eventHubName, String consumerGroupName) {
+        return Flux.fromIterable(new ArrayList<>(partitionOwnershipMap.values()));
+    }
+
+    @Override
+    public Flux<PartitionOwnership> claimOwnership(
+        List<PartitionOwnership> requestedPartitionOwnerships) {
+        return Flux.fromStream(requestedPartitionOwnerships.stream()
```

Comment:
Also, curious why the parameter is on a new line when the ones above it aren't. Our editorconfig is set to 120 characters, and this line isn't that long. 🤔 

use: `Flux.fromIterable(requestedPartitionOwnerships)`